easybuild/easyconfigs/n/numactl/numactl-2.0.11-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'numactl'
+version = '2.0.11'
```

Comment:
please update to 2.0.12 (see https://github.com/numactl/numactl/releases), and fix homepage too